docs/content/cookbook/oss-deployment-guide.md

```diff
@@ -0,0 +1,625 @@
+---
+layout: fullscreen
+hideFromSiteSearch: true
+---
+
+# OSS Deployment Guide
+
+In this guide we'll show you how to deploy an Open Source Vue Storefront application to a production server. 
+
+::tip
+This guide is for Open Source Applications. If you have Enterprise support, please refer to the [Cloud Deployment Guide](/cloud) or see the [Enterprise Onboarding Course](https://academy.vuestorefront.io/courses/enterprise-onboarding-course).
+::
+
+## Prerequisites
+::list{type="success"}
+- A linux server with SSH Access
+- A domain name and the ability to point it to your server's IP address
+- Remember to stay hydrated and get enough sleep 😉
+::
+
+## References
+::grid{:columns='3'}
+#section-1
+:card{to="https://github.com/rohrig/vsf-deployme-next" title="Reference Repo" description="See this guide live in action" icon="ri:book-2-fill"}
+
+#section-2
+:card{to="https://shoppernauts.com" title="Demo" description="A demo site deployed via this." icon="material-symbols:storefront"}
+::
+
+::tabs{:titles='["Prepare Server",  "Github Secrets and Variables","Deployment Script"]' class="mt-8"}
+
+#tab-1
+## Prepare Server
+
+### Update Server
+The command `apt-get update` is used to update the package index files on the system, which contain information about available packages and their versions.
+```bash
+apt-get update
+```
+
+### Install Node.js
+```bash
+curl -fsSL https://deb.nodesource.com/setup_18.x | bash - &&apt-get install -y nodejs
+```
+### ensure node is installed
+```bash
+node -v 
+```
+### Install yarn
+```bash
+npm install --global yarn
+```
+### ensure yarn is installed
+```bash
+yarn -v
+```
+
+### Install Nginx
```

Comment:
```suggestion
### Install nginx
```

Based on their [open source docs](https://nginx.org/en/docs/) - this seems to be the capitalization they use. But I've seen them also use all caps. But as long as it's consistent, it should be ok.